### PR TITLE
Enable list for RA/DEC input

### PR DESCRIPTION
**Pull request by profxj**
As titled.

Solely for the radec_to_coord() method.

No new docs.

### CODE REVIEW

File: linetools/utils.py [hunk 1]
```diff
@@ -143,6 +146,16 @@ def radec_to_coord(radec):
                 raise ValueError("radec must include + or - for DEC")
             newradec = (radec[0:2]+':'+radec[2:4]+':'+radec[4:sign+3] +':'+radec[sign+3:sign+5]+':'+radec[sign+5:])
             coord = SkyCoord(newradec, frame='fk5', unit=(u.hourangle, u.deg))
+    elif isinstance(radec,list):
+        clist = []
```
Comment (ntejos) [inline linetools/utils.py]: Does the list need to be tuples of coordinates in degrees?

Comment (profxj) [inline linetools/utils.py]: No

Comment (ntejos) [inline linetools/utils.py]: I see; I've missed the recursive bit. Looks good then.

**Comment by profxj** [comment]
Merging

**Pull request merged**